### PR TITLE
Create scatter plots

### PR DESCRIPTION
**Pull request by jethronap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,8 @@ Basic statistics and data analysis with Java
 - <a href="#">UseContinuousSampleExample</a> Show cases how to create a ```NumericSample```
 - <a href="#">LoadCsvDataSetWithTreeMap</a>: How to load  CSV data and put a column in a ```NumericSample```
 - <a href="#">MeanBootstrapExample</a>: Vanilla Bootstrap for the mean
+- <a href="#">PlotALine</a>: Plot a line
```
Comment (pockerman) [inline README.md]: Can you be a bit more verbose here? E.g. ```Shows how to plot a line blah blah blah....```

File: src/main/java/examples/PlotAScatter.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * Category: Plotting
+ * ID: PlotABar
+ * Description: Load a CSV file and plot a scatter chart with 3 numeric variables
+ */
+public class PlotAScatter {
+
+    public static void main(String[] args) throws IOException {
+        File file = new File("test_data/humans_data.csv");
```
Comment (pockerman) [inline src/main/java/examples/PlotAScatter.java]: Let's keep the ```test_data/``` only for the tests and use ```data/``` for the examples???

File: src/test/java/visualizations/ScatterChartTest.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
+package visualizations;
```
Comment (pockerman) [inline src/test/java/visualizations/ScatterChartTest.java]: same with this PR: https://github.com/jethronap/JavaDataAnalysis/pull/12 currently I don't think these tests put any value for us...the case is covered by the examples or no?

**Review by pockerman — CHANGES_REQUESTED** [review]
pls consider my comments

**Comment by pockerman** [comment]
merge the master branch into here so we don't see irrelevant changes

File: README.md (hunk 1 above)
Comment (jethronap) [reply README.md]: ok!

File: src/main/java/examples/PlotAScatter.java (hunk 2 above)
Comment (jethronap) [reply src/main/java/examples/PlotAScatter.java]: updated file path for all cases

File: src/test/java/visualizations/ScatterChartTest.java (hunk 3 above)
Comment (jethronap) [reply src/test/java/visualizations/ScatterChartTest.java]: removed tests.

**Comment by jethronap** [comment]
pls check out the updates so that we can merge.

**Pull request merged**